model/action_test.go

```diff
@@ -0,0 +1,551 @@
+package model
```

Comment:
https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide#8-license-header-in-each-go-file